participants/philip-saa.json

```
@@ -0,0 +1,28 @@
+{
+	"realName": {
+		"givenName": "Philip",
+		"familyName": "Saa",
+		"placeFamilyNameFirst": false,
+		"hideFamilyNameOnWebsite": false
+	},
+	"githubAccountName": "cowglow",
+	"company": "Saab Deutschland GmbH",
+	"when": {
+		"friday": true,
+		"saturday": true
+	},
+	"iCanTakeNotesDuringSessions": false,
+	"tags": ["Indie Web", "A/V", "React", "Svelte", "TypeScript", "Playwright"],
+	"vegan": false,
+	"vegetarian": true,
+	"allergies": [],
+	"whatIsMyConnectionToJavascript": "I joined the internet in 1996 and I am currently working with JavaScript/TypeScript",
+	"whatCanIContribute": "I can help you with hurdles you might have when it comes to creating audio/video content.",
+	"tShirtSize": {
+		"size": "3XL",
+		"type": "regular"
+	},
```

That's still the old format (kind of),

```suggestion
	"tShirtSize": "3XL",
```